docs/build/spells.md

```diff
@@ -0,0 +1,197 @@
+# Spells
+
+Spells are actors in the Fluence Network designed to perform tasks with user business logic as well as maintenance jobs. 
+
+Think of them as of automated jobs, that can be executed when some specific trigger event occurs (time, connection poll, http, blockchain events, etc).
+
+Spells can be used for user-defined automation scenarios for service maintenance, periodical data extraction, subnet maintenance, etc. 
+
+A spell is a [Marine](/docs/marine-book/introduction.md) service, the main goal of which is to provide state storage for script execution. It also provides some additional features to ease the spell usage for users and take some responsibility from the node (aka [Nox](https://github.com/fluencelabs/nox)). The service serves as a template and is instantiated on the creation of a new spell.
```

Comment:
It should be mentioned that Spells contain and execute AIR scripts.

Otherwise, it reads like "jobs" are defined in Rust, in WASM.